### ISSUE TITLE
where check logo part?

### ISSUE THREAD

**Issue by Toei79** [K-vanc/Tempest-EPG-Generator#23]
just install latest version feels faster that previous one that already 

how check the logo part? idk where find it,  i check manual  i see version number its previous one sorry idk where check,

i update replacing php file  and config.yml 

i was thinking its in tempest configurator>channel indexer part or its other section?

**Comment by K-vanc** [comment]
hi. you just need to download and replace tempest.php file to update. config.yml is a github file and it is not related with Tempest.

Probably you have multiple copies and still calling/running the old copy. Check your command line path for location of tempest.php, go that directory, delete tempest.php and paste new one..

That issue was for old version. As of rv1.5.0, you can set your channel logos from Tempest Configurator -> Channel Indexer by adding ||#logo# command with url of your logo to end of Channel's site_id as below;

![image](https://user-images.githubusercontent.com/97025515/227836675-3ad8c1ba-76c8-4615-87e4-8c02afa3f4c7.png)

**Comment by Toei79** [comment]
oh its there in site id , on channel indexer,  it can be on any server, just saying i use google for mine, and because large links

i have one copy in html, folder then other copy in a folder but i get with link http://192.168.2.131/tempest.php like this i deleted file and reboot still getting the same old version despite i replace last one.   im guessing its the old version because when i run script on command line i see the 1.4.2 ,  but i need recheck usually its in my side something wrong.

**Comment by K-vanc** [comment]
##cclogo## is part of siteconfig file and can be set only from inside of siteconfig which you cannot for [ENC] ones. New ||#logo# command is allowing you to change/set any channel logo regardless of siteconfig is unlocked or encrypted. No matter link is large or not, you can add to end of site_id line as shown. It will work as long as link is reachable. You can also use some free url shorteners if you want

**Comment by Toei79** [comment]
thanks now i need figure out version file because i did, i delete file and try and it was unreachable, then i reboot and paste still getting 1.4.2 on command line , if its 1.5 if i run command line epg generator , its gonna shows 1.5 there? im gonna check and see thanks again

**Comment by K-vanc** [comment]
yes, you should see rv1.5.0 on both cli and gui mode. To be sure, Just delete all tempest.php files in your system including the one you downloaded then redownload from repo and paste into root

**Comment by Toei79** [comment]
ok if i paste permissons set to root its fine? i did www data user. im going to do that thanks

**Comment by K-vanc** [comment]
or for verification download new one from repo and rename it something like tempest123.php. Paste it to root and call tempest123.php to be sure. You can rename tempest.php as you want since file name set as dynamic as a security for remote access. 

Yes, after copy/paste, it is better to check permissions/ownerships. Sometimes file ownerships may be taken by root user without your request

**Comment by Toei79** [comment]
![image](https://user-images.githubusercontent.com/79808610/227848327-27802d06-d075-4106-8b01-d2957b10679e.png)

**Comment by Toei79** [comment]
ok! i found it

the php file alone its last version 1.5

but the one in the zip file its the older one.

**Comment by K-vanc** [comment]
ah, I see now. When I make a release, github is zipping all files available at that moment. I usually make a release than I am replacing individual files on main folder directory. Next time, I will replace main folder files first and make the release after so zip file will contain also the latest

**Comment by Toei79** [comment]
they do something like compare files vs olds ones or some like that can helps.

**Comment by K-vanc** [comment]
This one actually was my fault. This is my first project in github so still learning how things are happening in github including releases. Also for a while, i was  thinking, people are getting latest siteconfigs by clicking download project as zip but found out that you can only download the project copy on release moment as zip then coded Siteconfig Updater on rv1.5.0.
